server/routes/shared/riskView.ts

```diff
@@ -0,0 +1,62 @@
+import { DetailsArgs, TableArgs, TagArgs } from '../../utils/govukFrontendTypes'
+import utils from '../../utils/utils'
+import RiskPresenter from './riskPresenter'
+
+export default class RiskView {
```

Comment:
Do you think it might be good to have some way to separate classes which are used to render a page (i.e. extend PageContentView) and classes which are used for composition of page rendering classes?

I had a similar situation with addresses and set it as a Component class rather than View class `addressFormComponent.ts`

Reply:
yeh that's a good idea. we have a few like this so i think we should get together and come up with a pattern we can apply across all the shared view / presenter classes.